lib/fizzy/execute.cpp

```diff
@@ -362,6 +375,25 @@ inline double fneg(double value) noexcept
     return bit_cast<double>(bit_cast<uint64_t>(value) ^ F64SignMask);
 }
 
+template <typename T>
+T fcopysign(T a, T b) noexcept = delete;
+
+template <>
```

Comment:
Didn't add this, just moved this function. Can remove.

Reply:
Still good time to improve it.

Reply:
> These should not be template specializations. Function overloading is good enough.

Actually, I've learned this is not enough because of the way this is used. Ignore this request.

Reply:
Yes, learned that yesterday too, but was lazy to close the PR 😓